tests/serialized_test_data_generation/Makefile

```diff
@@ -68,14 +74,18 @@ run_model: ## run model using Docker image to generate serialized data
 	docker run -e USER_ID_HOST=$(USER_ID_HOST) -e GROUP_ID_HOST=$(GROUP_ID_HOST) \
 		-e SER_ENV=$(SER_ENV) \
 		--network host --rm -v $(RUN_DIR_HOST):$(RUN_DIR_CONTAINER) \
-		-v $(DATA_DIR_HOST):$(DATA_DIR_CONTAINER) $(COMPILED_IMAGE) \
-		/rundir/submit_job.sh
+		-v $(DATA_DIR_HOST):$(DATA_DIR_CONTAINER) \
+                --mount type=tmpfs,destination=$(RUN_DIR_CONTAINER)/test_data \
+                --shm-size=512m \
+		$(COMPILED_IMAGE) /rundir/submit_job.sh
 	cd $(DATA_DIR_HOST) && \
-		md5sum `/bin/ls -1d * | egrep -v '^fortran_sha.txt$$|^logfile.000000.out$$|^stdout.out$$|^stderr.out$$|^env.out$$'` > md5sums.txt
+		find . -type f \( -name "*.dat" -o -name "*.nml" -o -name "*.json" \) -exec md5sum {} \; > md5sums.txt
```

Comment:
Yeah, there was a tradeoff here: The current implementation did not work for the 54 rank case on my local Mac because of the command line becoming too long with the `/bin/ls -1d *`. If there are better solutions and we can retain the current logic, I am a taker!